assignment5/src/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import { BiUserCircle } from "react-icons/bi";
+import SearchBar from "./SearchBar";
+
+const Header = (props) => {
+  const { setDeveloperData, userData } = props;
+  return (
+    <div className="header">
+      <div className="logo">Storm Troopers</div>
+      <SearchBar setDeveloperData={setDeveloperData} userData={userData} />
```

Comment:
It will be good to keep search bar outside header container.
That way if some other page doesn't contains search bar we wont have to rewrite our code.
Header and footer should have static data. :)